cxbox-core/src/main/java/org/cxbox/core/dao/impl/MetadataUtils.java

```diff
@@ -269,8 +271,16 @@ public static Predicate createPredicate(Root<?> root, ClassifyDataParameter crit
 								.map(object -> cb.equal(cb.upper(field), requireString(object).toUpperCase()))
 								.toArray(Predicate[]::new));
 					} else {
-						return cb
-								.or(((List<Object>) value).stream().map(object -> cb.equal(field, object)).toArray(Predicate[]::new));
+						String[] split = criteria.getField().split("\\.");
```

Comment:
we need description here with example (what cxbox project code is converted to what sql) - write it in java doc. extract to separate method to have place for java doc